_source/logzio_collections/_log-sources/fluentd-docker.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Ship container logs with Fluentd on Docker
+logo:
+  logofile: FluentD_Docker.png
+  orientation: vertical
+short-description: Ship container logs with Fluentd on Docker
+data-source: Fluentd on Docker
+shipping-tags:
```

Comment:
Can you add a reference to our os project, like in this doc:
![chrome-capture (5)](https://user-images.githubusercontent.com/48534558/123853378-1046eb80-d926-11eb-8627-e8c46185693d.jpg)
[The doc on github](https://github.com/logzio/logz-docs/blob/master/_source/logzio_collections/_metrics-sources/aws-cloudfront.md)

[Github repo](https://github.com/logzio/fluentd-docker-logs)